pkg/operator/ceph/cluster/crash/keyring.go

```diff
@@ -37,8 +38,8 @@ const (
 )
 
 // CreateCrashCollectorSecret creates the Kubernetes Crash Collector Secret
-func CreateCrashCollectorSecret(context *clusterd.Context, namespace string, ownerRef *metav1.OwnerReference) error {
-	k := keyring.GetSecretStore(context, namespace, ownerRef)
+func CreateCrashCollectorSecret(context *clusterd.Context, clusterInfo *client.ClusterInfo, ownerRef *metav1.OwnerReference) error {
```

Comment:
Ditto, can we remove `ownerref` from the signature since we have it in `clusterInfo`?